### PR TITLE
Error message for wrong semester

### PR DESCRIPTION
**Pull request by VictorLi718**
screen shot 2018-10-05 at 16 42 51
I have changed the error message for students entering wrong semester like this.

### CODE REVIEW

**Comment by VictorLi718** [comment]
image

File: site/app/libraries/Core.php [hunk 1]
```diff
@@ -426,6 +426,20 @@ public function getDisplayedCourseName(){
...
         }
     }
+
+    public function getFullSemester(){
+        $semester = $this->getConfig()->getSemester();
+        if ($this->getConfig()->getSemester() !== ""){
+            $arr1 = str_split($semester);
+            $semester = "";
+            if($arr1[0] == "f")  $semester .= "Fall ";
+            else if($arr1[0] == "s")  $semester .= "Spring ";
```
[the remark this answers is not in the document]
Comment (VictorLi718) [reply site/app/libraries/Core.php]: Sorry, I don't know what the summer semester start with..

[the remark this answers is not in the document]
Comment (MasterOdin) [reply site/app/libraries/Core.php]: Sorry, I should have specified to add that line in my comment to your code to handle summer.

File: site/app/libraries/Core.php [hunk 2]
```diff
@@ -426,6 +426,20 @@ public function getDisplayedCourseName(){
...
+
+            $semester .= "20". $arr1[1]. $arr1[2];
+
+            return $semester;
```
[the remark this answers is not in the document]
Comment (VictorLi718) [reply site/app/libraries/Core.php]: My bad. Should be outside.

**Pull request merged**